source/assets/stylesheets/all.scss

```diff
@@ -17,6 +17,22 @@
 @import "components/portfolio";
 @import "components/rubyshop";
 
+#events {
+  .timeline-image {
```

Comment:
Its unnecessary to add specificity if timeline-image is only used in events context. 
No code is better than more code